packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -385,4 +391,75 @@ class VideoPlayerOptions {
   /// Note: This option will be silently ignored in the web platform (there is
   /// currently no way to implement this feature in this platform).
   final bool mixWithOthers;
+
+  /// Additional web controls
+  final VideoPlayerWebOptions? webOptions;
+}
+
+/// [VideoPlayerWebOptions] can be optionally used to set additional web settings
+@immutable
+class VideoPlayerWebOptions {
+  /// [VideoPlayerWebOptions] can be optionally used to set additional web settings
+  const VideoPlayerWebOptions({
+    this.controls = const VideoPlayerWebOptionsControls.disabled(),
```

Comment:
The current default behavior of the `video_player` package on web is to not show the web controls:

<img width="1108" alt="Bildschirmfoto 2023-07-07 um 10 02 18" src="https://github.com/flutter/packages/assets/13286425/f4920c46-5d2b-4fbb-892d-6f3b39aca13d">

If we default to `.enabled`, it will automatically be enabled for all projects which use `video_player`.

Reply:
Ah yes, you're right, controls are disabled here:

* https://github.com/flutter/packages/blob/3eaad3d0d1ecbf8425b72a28b150c013781ee4fa/packages/video_player/video_player_web/lib/src/video_player.dart#L60-L63

Shame that the API never allowed passing any configuration to the initialization of the video player. What a weird choice in hindsight!